server/src/dotConfig.mjs

```diff
@@ -58,6 +58,30 @@ function loadDotConfig() {
   return dotConfig;
 }
 
+/* 
+* @function:findFileCreationAndModificationTime
+* @Description: To get creation and modification time of files
+@param {String} creationDateFileName - Name of file whose creation time needs to be retrieved in seconds
+@param {String} modificationDateFileName - Name of file whose modification time needs to be retrieved in seconds
+* @Return: Array containing creation and modification time in seconds ex: [1687860231, 1687860231]
+*/
+
+function findFileCreationAndModificationTime(creationDateFileName, modificationDateFileName) {
+  let cFile, mFile, __dirname, __filename
+  __filename = fileURLToPath(import.meta.url).replace("build", "src");
+  __dirname = path.dirname(__filename);
+  cFile = path.resolve(__dirname, creationDateFileName);
+  mFile = path.resolve(__dirname, modificationDateFileName);
+  const creationTimeSec = Math.floor(fs.statSync(cFile).birthtimeMs / 1000);
+  const modificationTimeSec = Math.floor(fs.statSync(mFile).mtimeMs / 1000);
+  return [creationTimeSec, modificationTimeSec]
+}
+
+const [creationTimeSec, modificationTimeSec] = findFileCreationAndModificationTime('.mf.config.SAMPLE.json', '.mf.config.json')
+if (creationTimeSec >= modificationTimeSec) {
+  logger.importantWarning(`Refer release notes to check for new/modified configs. You probably want to "cp src/.mf.config.SAMPLE.json src/.mf.config.json && npm run build:mf"`)
```

Comment:
Small grammar fix to the message:

The .mf.config.SAMLE.json file in your repo is newer than your .mf.config.json file. Changes to the config file format may have occurred since you created or edited your file, and you may need to merge your changes within the changes to the sample file. Refer to the release notes for more information.

Note: I would _not_ include the "You probably want to ..." because if they've edited their file, this would nuke it.

Reply:
Seconded

Reply:
Addressed